pkgs/development/interpreters/cling/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, python, wget, fetchFromGitHub, libffi, cacert, git, cmake, llvm, ncurses, zlib, fetchgit, glibc, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "cling";
+  version = "2019-07-05";
+
+  clingSrc = fetchFromGitHub {
+    owner = "root-project";
+    repo = "cling";
+    rev = "5789f5b2eb14ff8e14e6297fe3a449d0ef794f17";
+    sha256 = "06l0c9p8102prhanhv8xawzq4k8g587d46aykc2y5j7rrzsbs5hs";
+  };
+
+  clangSrc = fetchgit {
+    url = "http://root.cern.ch/git/clang.git";
+    rev = "dd71e0397cfc8667d4c75ef5b4a1b35802608e59"; # This is the head of the cling-patches branch
+    sha256 = "1ir1s4wy9683p9wrvra4xb4xqcd0amqnm4wahx1pf72zk2yxw1wh";
+  };
+
+  llvmSrc = fetchgit {
+    url = "http://root.cern.ch/git/llvm.git";
+    rev = "e0b472e46eb5861570497c2b9efabf96f2d4a485"; # This is the head of the cling-patches branch
+    sha256 = "0yls35vyfcb14wghryss9xsin4cbpgkqckg72czh5jd2w0vjcmbx";
+  };
+
+  srcs = [clingSrc llvmSrc clangSrc];
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    cp -r "$llvmSrc/." llvm/
+    chmod u+w llvm/tools
+    mkdir -p llvm/tools/cling && cp -r "$clingSrc/." llvm/tools/cling
+    mkdir -p llvm/tools/clang && cp -r "$clangSrc/." llvm/tools/clang
+
+    runHook postUnpack
+  '';
+
+  buildInputs = [python wget cacert git cmake llvm libffi makeWrapper];
```

Comment:
`makeWrapper` needs to go to `nativeBuildInputs`. `git` and `python` probably also need to go to `nativeBuildInputs`.
You probably don't need `wget` or `cacert` to build this. It's a bit strange that you need to depend on `llvm` here.

Reply:
Of course `cmake` goes to `nativeBuildInputs` too.